tests/sample/sample.vcxproj.filters

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I can't tell, what is the change in this line?

Reply:
There are many such lines that the tool is showing as diff, but there arent any changes - not even white spaces. Not sure what to do about these. So I kept them.

Reply:
If there are no changes, you remove them from the changelist.

Reply:
@dthaler all of these are removing the [BOM](https://en.wikipedia.org/wiki/Byte_order_mark). I could ignore these changes or keep them. Preference?

Reply:
I have removed this and other files with only BOM changes.

Reply:
Actually I'd say remove the BOM changes since they're not part of "Fuzzing bug fixes".